src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TargetElementAttribute.cs

```diff
@@ -41,14 +41,17 @@ public TargetElementAttribute(string tag)
         /// <summary>
         /// The HTML tag the <see cref="ITagHelper"/> targets.
         /// </summary>
-        /// <remarks>A <c>*</c> <see cref="Tag"/> value indicates an <see cref="ITagHelper"/> 
+        /// <remarks>A <c>*</c> <see cref="Tag"/> value indicates an <see cref="ITagHelper"/>
         /// that targets all HTML elements with the required <see cref="Attributes"/>.</remarks>
         public string Tag { get; }
 
         /// <summary>
-        /// A comma-separated <see cref="string"/> of attributes the HTML element must contain for the 
+        /// A comma-separated <see cref="string"/> of attribute names the HTML element must contain for the
         /// <see cref="ITagHelper"/> to run.
         /// </summary>
+        /// <remarks>
+        /// <c>*</c> at the end of an attribute name acts as a wild card.
```

Comment:
On second look, I'm not sure the "wild card" mention helps. The asterix changes the `Attributes` entry to a prefix match, nothing as general as a wildcard.